python/ray/serve/scripts.py

```diff
@@ -114,9 +129,68 @@ class may or may not be decorated with ``@serve.deployment``.
     type=str,
     help="JSON string for the deployments options",
 )
-def deploy(deployment: str, options_json: str):
+def create_deployment(deployment: str, options_json: str):
     deployment_cls = import_attr(deployment)
     if not isinstance(deployment_cls, Deployment):
         deployment_cls = serve.deployment(deployment_cls)
     options = json.loads(options_json)
     deployment_cls.options(**options).deploy()
+
+
+@cli.command(
+    help="""
+    [Experimental] Deploy a YAML configuration file via REST API to
+    your Serve cluster.
+    """,
+    hidden=True,
+)
+@click.argument("config_file_name")
+@click.option(
+    "--address",
+    "-a",
+    default=os.environ.get("RAY_ADDRESS", "http://localhost:8265"),
+    required=False,
+    type=str,
+    help='Address of the Ray dashboard to query. For example, "http://localhost:8265".',
+)
+def deploy(config_file_name: str, address: str):
+    full_address_path = f"{address}/api/serve/deployments/"
+
+    with open(config_file_name, "r") as config_file:
+        config = yaml.safe_load(config_file)
+
+    # Generate a schema using the config to ensure its format is valid
+    ServeInstanceSchema.parse_obj(config)
+
+    response = requests.put(full_address_path, json=config)
+
+    if response.status_code == 200:
+        print(
```

Comment:
use logger not print

Reply:
see the shared Ray CLI logger we use in the jobs cli `dashboard/modules/job/cli.py`

Reply:
Fixed.